### PR TITLE
Restore MediaWiki 1.35(.3+) compatibility

### PR DESCRIPTION
**Pull request by mary-kate**
This could be done rather easily with literally one simple `if()`, so why not?
After all, the 1.35 release is the current Long-Term Support (LTS) release so it makes sense to support it until the next LTS becomes available, given that Liberty is not using anything else that's strictly available only in MW 1.36+.

### CODE REVIEW

**Comment by zlzleking** [comment]
Already we have REL1_35 branch and it is currently maintained (librewiki uses this branch for production enviroment). and if next mediawiki LTS version is rolled out. librewiki will update 'mediawiki' and make release branch (eg REL1_39). 

The master branch reflects the latest developments and aims to keep up with only the latest version of MediaWiki.

other functional changes dosesn't need on legacy enviroment (eg. LTS). so i manually backport functions (eg. visualeditor related css fix). 

In conclusion, we don't have any plan to roll out the latest changes for legacy version of mediawiki (on master branch).
